### ISSUE TITLE
--additional-view option for anvi-interactive

### ISSUE THREAD

**Issue by meren** [merenlab/anvio#160]
Assume you have a profile database, and an extra matrix file of values for each split in each sample that appears in that profile database. Currently one way to do it is to use `-f`, `-m`, and `-t` parameters to start an ad hoc interactive interface where you can visualize everything. But this method does not allow you to enjoy all the other bells and whistles such as being able to make good selections and summarizing them, or inspecting splits.

Long story short, we need an `--additional-view` parameter for anvi-interactive.
